lib/ttfunk/subset/code_page.rb

```diff
@@ -0,0 +1,82 @@
+require 'set'
+
+require_relative 'base'
+
+module TTFunk
+  module Subset
+    class CodePage < Base
+      class << self
+        def unicode_mapping_for(encoding)
+          mapping_cache[encoding] ||= (0..255).each_with_object({}) do |c, ret|
+            # rubocop:disable Lint/HandleExceptions
+            begin
+              ret[c] = c.chr(encoding)
+                        .encode(Encoding::UTF_8)
+                        .codepoints
+                        .first
+            rescue Encoding::UndefinedConversionError
+              # Ruby doesn't appear to think there is a strict 1:1 mapping
```

Comment:
This comment is not quite accurate. Ruby's opinion has nothing to do with convertability. There're actually illegal coversions.

For example, `0x81` is undefined in CP1252. There's no way to convert it into Unicode.

So maybe just say that there's no 1:1 mapping between all encodings and Unicode.

Reply:
Ok, that's something I hadn't realized, thanks for the explanation.